posthog/models/async_migration.py

```diff
@@ -11,6 +11,13 @@ class MigrationStatus:
     Starting = 5  # only relevant for the UI
 
 
+class AsyncMigrationError(models.Model):
+    id: models.BigAutoField = models.BigAutoField(primary_key=True)
+    created_at: models.DateTimeField = models.DateTimeField(auto_now_add=True, blank=True)
+    description: models.CharField = models.CharField(max_length=400, null=False, blank=False)
```

Comment:
is "description" the actual error? not sure this is a suitable name

Reply:
open to naming suggestions :) "message" sounded odd